R/utils.R

```diff
@@ -34,6 +34,31 @@ WSTextWriter <- function(bufferSize=1024){
   # TRUE if we're eating whitespace right now, in which case calls to writeWS are no-ops.
   suppressing <- FALSE
 
+  # Collapses the text in the buffer to create space for more writes. The first
+  # element in the buffer will be the concatenation of any writes up to the
+  # current marker. The second element in the buffer will be the concatenation
+  # of all writes after the marker.
+  collapseBuffer <- function(){
```

Comment:
Can you add a space between `function()` and `{`? Sorry, that kind of stuff drives me crazy.

Reply:
I smell an Rlint project for me to work on... ;)